parsedown

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env php
+<?php
+
+require(dirname(__FILE__).'/Parsedown.php');
+
+// From a file
+if (isset($argv[1])) {
+
+	if ( ! is_readable($argv[1])) {
+		throw new RuntimeException(sprintf(
+			'Could not read "%s"',
+			$argv[1]
+		));
```

Comment:
I thought of that, but I'd like users to see what's the actual exception. And I thought throwing an exception in a PHP CLI would output it and exit with non-zero status code. But I need to verify that actually. Perhaps an option should be enabled. I'll think about that, thanks!